ansible/doc/README.testbed.VsSetup.md

```diff
@@ -111,13 +112,16 @@ vm_host_1
 
 - Add user ```foo```'s private key to ```$HOME/.ssh/id_rsa``` inside sonic-mgmt docker container.
 
-- Add user ```foo```to sudoer list, use ```visudo``` to add following line in the sudoer configuration.
+- Add user ```foo``` to sudoer list, use ```visudo``` to add following line in the sudoer configuration on the host.
+
 ```
    foo ALL=(ALL) NOPASSWD:ALL
 ```
 
+- Make sure user ```foo```'s home directory is owned by user ```foo```. If necessary, ```chown -R foo:foo /home/foo``` on the host.
```

Comment:
Only need single backticks for inline code block. Triple backticks are necessary for multi-line blocks.